app/views/application/head/_theme.html.erb

```diff
@@ -0,0 +1,14 @@
+<% if defined?(Current.user.preferences[:color_theme]) && Current.user.preferences[:color_theme].present? %> 
+    <script>
+        document.documentElement.setAttribute('data-theme', '<%= Current.user.preferences[:color_theme] %>');
+    </script>
+<% else %>
+    <script>
+        const prefersDarkScheme = window.matchMedia('(prefers-color-scheme: dark)');
+        if (prefersDarkScheme.matches) {
+            document.documentElement.setAttribute('data-theme', 'dark');
+        } else {
+            document.documentElement.setAttribute('data-theme', 'light');
+        }
+    </script>
+<% end %>
```

Comment:
How about we just use a class, the default config, rather than a data attribute?

Based on my quick read of the docs you linked to (https://tailwindcss.com/docs/dark-mode#toggling-dark-mode-manually) I think you just change the tailwind setting to be darkmode `selector`

So my suggestion is:
* Alter the darkMode below to just be `selector` with no data key after it
* Change this bit of javascript to do `document.documentElement.classList.add('dark')` or remove()

It feels just a little simpler for it to be class